death-clock.js

```diff
@@ -113,11 +116,22 @@ const tickTock = () => {
 getBirthdate();
 
 let target;
+
+//declaring here to not have it tick.
+const youtubeDarkMode = socialMediaPlatform == 'youtube' && document.documentElement.getAttribute("dark");
+
 const findTarget = setInterval(() => {
   target = document.querySelector(socialMediaTargetClasses[socialMediaPlatform]);
   if (target) {
-    target.id = `death-clock-${socialMediaPlatform}`;
-
+    
+    //Support for youtube darkmode. Changes font. Leaves open to creating more darkmodes in future.
+    if(youtubeDarkMode){
+        target.id = `death-clock-${socialMediaPlatform}-dark`;
```

Comment:
I feel like it would be cleaner if we use `dark` and [add it to the class list](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList). That way we can just target the `.dark` class. That way the `target.id` will always be the same.